### PR TITLE
Fix pom files for releases

### PR DESCRIPTION
**Pull request by lankavitharana**
## Purpose
Fix pom issues so that release build works without any issue

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -748,6 +748,34 @@
                     </plugin>
                 </plugins>
             </build>
+            <modules>
+                <module>bvm/ballerina-core</module>
+                <module>bvm/ballerina-logging</module>
+                <module>cli/ballerina-launcher</module>
+                <module>cli/ballerina-packerina</module>
+                <module>compiler/ballerina-lang</module>
```
Comment (sameera-jayasoma) [inline pom.xml]: The compiler module should be the first module here.  Even though Maven reorders this, it is clean if we order it manually.

Comment (lankavitharana) [reply pom.xml]: I did it in ascending order, :) will change and update

Comment (sameera-jayasoma) [reply pom.xml]: I see :) 

It looks cleaner if the module order in this pom.xml is same as the actual dependency order.

Comment (lankavitharana) [reply pom.xml]: changed

**Pull request merged**